rgw/v2/tests/s3_swift/test_bucket_listing.py

```diff
@@ -286,15 +288,16 @@ def test_exec(config, ssh_con):
             reusable.get_radoslist()
 
         if config.test_ops.get("delete_bucket_object", False):
-            if config.test_ops.get("enable_version", False):
-                for name, path in objects_created_list:
-                    reusable.delete_version_object(
-                        bucket, name, path, rgw_conn, each_user
-                    )
-            else:
-                reusable.delete_objects(bucket)
-                time.sleep(30)
-                reusable.delete_bucket(bucket)
+            for bkt in bucket_created:
+                if config.test_ops.get("enable_version", False):
+                    for name, path in objects_created_list:
+                        reusable.delete_version_object(
+                            bkt, name, path, rgw_conn, each_user
+                        )
```

Comment:
@anrao19 , we are not removing bucket here right!

Reply:
@hmaheswa, deletion of bucket will be performed for both versioned and non-versioned bucket